### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by LordRatte**
It's very easy for corner cases to get missed so I added a few basic ones to make the project more robust.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -38,3 +38,31 @@ pub fn match_increase_or_decrease(first: u32, second: u32) -> (String, f32) {
...
+
+#[cfg(test)]
+mod tests {
+    use super::{
+        calculate_percentage_decrease, calculate_percentage_increase, match_increase_or_decrease,
+    };
+    #[test]
+    fn test_calculate_percentage_increase() {
+        assert_eq!(calculate_percentage_increase(1, 2), 100.0);
+    }
```
Comment (thomaschaplin) [inline src/lib.rs]: Might as well add the increase of 0.0 here too
```suggestion
    fn test_calculate_percentage_increase() {
        assert_eq!(calculate_percentage_increase(1, 2), 100.0);
        assert_eq!(calculate_percentage_increase(1, 1), 0.0);
    }
```

File: src/lib.rs [hunk 2]
```diff
@@ -38,3 +38,31 @@ pub fn match_increase_or_decrease(first: u32, second: u32) -> (String, f32) {
         Ordering::Equal => (EQUAL.to_string(), 0.0),
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::{
+        calculate_percentage_decrease, calculate_percentage_increase, match_increase_or_decrease,
+    };
```
Comment (thomaschaplin) [inline src/lib.rs]: Thinking it's worth importing all as my other comment suggests using `INCREASE`, `DECREASE` and `EQUAL`
```suggestion
    use super::*;
```

File: src/lib.rs [hunk 3]
```diff
@@ -38,3 +38,31 @@ pub fn match_increase_or_decrease(first: u32, second: u32) -> (String, f32) {
...
+    #[test]
+    fn test_calculate_percentage_increase() {
+        assert_eq!(calculate_percentage_increase(1, 2), 100.0);
+    }
+
+    fn test_calculate_percentage_decrease() {
+        assert_eq!(calculate_percentage_increase(2, 1), 100.0);
+    }
```
Comment (thomaschaplin) [inline src/lib.rs]: Missing the `#[test]` and calling the wrong function. Plus the right hand side value was `100.0` when it should be `50.0`. Would be nice to add the second assert too
```suggestion
    #[test]
    fn test_calculate_percentage_decrease() {
        assert_eq!(calculate_percentage_decrease(2, 1), 50.0);
        assert_eq!(calculate_percentage_decrease(1, 1), 0.0);
    }
```

File: src/lib.rs [hunk 4]
```diff
@@ -38,3 +38,31 @@ pub fn match_increase_or_decrease(first: u32, second: u32) -> (String, f32) {
...
+        let symbol_minus = "-";
+        assert_eq!(
+            match_increase_or_decrease(2, 1),
+            (symbol_plus.to_string(), 100.0)
+        );
+        assert_eq!(
+            match_increase_or_decrease(1, 2),
+            (symbol_minus.to_string(), 100.0)
+        );
+    }
```
Comment (thomaschaplin) [inline src/lib.rs]: Missing the `#[test]` and would also be nice to add the third assert here too. Along with using the already declared chars
```suggestion
    #[test]
    fn test_match_increase_or_decrease() {
        assert_eq!(
            match_increase_or_decrease(2, 1),
            (DECREASE.to_string(), 50.0)
        );
        assert_eq!(
            match_increase_or_decrease(1, 2),
            (INCREASE.to_string(), 100.0)
        );
        assert_eq!(match_increase_or_decrease(1, 1), (EQUAL.to_string(), 0.0));
    }
```

**Review by thomaschaplin — CHANGES_REQUESTED** [review]
Wow, thank you so much for adding these tests!

I've raised a couple of suggestions, please take a look at my comments

**Review by thomaschaplin — APPROVED** [review]
I applied the suggestions, hope you don't mind? 

Thanks again for the PR!
